modules/ROOT/pages/dataweave-type-selector.adoc

```diff
@@ -0,0 +1,209 @@
+= Type Selector
```

Comment:
do we think of this as a type selector or a single-value selector that you can use for other data structures?